templates/ethernet_Debian.j2

```diff
@@ -1,4 +1,9 @@
+# {{ ansible_managed }}
+
 {% if item.bootproto == 'static' %}
+{% if item.pre_up is defined %}
+pre-up {{ item.pre_up }}
+{% endif %}
```

Comment:
Placing this here means if bootproto is DHCP then pre-up won't work. You could move before the first 'if', or duplicate on L32.

Also, does it make sense to also support this for bonds and bridges?

Reply:
I put the option before the ip/proto.

Regarding the man, this option is explained in the iface options with this message `The  following  "command" options are available for every family and method.`
In this section we have pre-up, up, post-up, down, pre-down, post-down and description.

Tell me if you want me to add more options, in other files...

Reply:
I think at least it should be consistent for ethernet, bridge and bond interfaces. If you want to add the other command options, feel free.